Documentation~/com.unity.xrlinerenderer.md

```diff
@@ -0,0 +1,30 @@
+# XR Line Renderer
+An XR-Optimized line renderer that is also capable of producing very inexpensive glow effects.  The XRLineRenderer mimics rendering with 3d capsules while only using two quads worth of geometry.
+
+## Setup and usage
+
+1. Place the XRLineRenderer folder into Assets\XR Utilities\XRLineRenderer in your project.
+
+2. Add a XRLineRenderer or XRTrailRenderer component to your gameobject.  The interface is nearly identical to the built in Unity Line and Trail Renderers.
+
+3. Create a new material using the XRLineRenderer shaders.  You can find some examples in XRLineRenderer\Materials
+
+4. Apply this material to the mesh renderer of your XRLineRenderer or XRTrailRenderer.
+
+
+## VRLineRenderer Shader
```

Comment:
@AndrewTHEManeri should we switch all the `VR` to `XR`?

Reply:
Yah